src/main/webapp/app/shared/links/TextWithRefs.tsx

```diff
@@ -0,0 +1,47 @@
+import { REFERENCE_LINK } from 'app/config/constants/constants';
+import { ParsedRef, parseReferences } from 'app/oncokb-commons/components/RefComponent';
+import React from 'react';
+
+export interface ITexWithRefsProps {
+  content: string;
+}
+
+/* eslint-disable @typescript-eslint/prefer-regexp-exec */
```

Comment:
Do we need this?

Reply:
Yes, since the component uses `Regex.match`

Reply:
Based on the check, wouldn't be using exec better?

Reply:
I think either way works, so I changed it to `exec` so we don't need to disable eslint